pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotSegmentRestletResource.java

```diff
@@ -622,7 +622,8 @@ public SuccessResponse reloadSegmentDeprecated2(
   public ServerReloadControllerJobStatusResponse getReloadJobStatus(
       @ApiParam(value = "Reload job id", required = true) @PathParam("jobId") String reloadJobId)
       throws Exception {
-    Map<String, String> controllerJobZKMetadata = _pinotHelixResourceManager.getControllerJobZKMetadata(reloadJobId);
+    Map<String, String> controllerJobZKMetadata = _pinotHelixResourceManager.
+            getControllerJobZKMetadataResource(reloadJobId, ControllerJobType.RELOAD_SEGMENT.name());
```

Comment:
This should reference both ControllerJobType.RELOAD_SEGMENT as well as ControllerJobType.RELOAD_ALL_SEGMENTS

Reply:
I've used RELOAD_SEGMENT as the jobType for both the cases (reload, reload_all). However I've retained RELOAD_ALL_SEGMENTS as a valid jobType for now as the UI (reload status) depends on that parameter. Once UI is updated, I can remove RELOAD_ALL_SEGMENTS.